backend/src/main/java/net/ugurkartal/backend/controllers/OrderDetailController.java

```diff
@@ -0,0 +1,30 @@
+package net.ugurkartal.backend.controllers;
+
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import net.ugurkartal.backend.services.abstracts.OrderDetailService;
+import net.ugurkartal.backend.services.dtos.requests.OrderDetailRequest;
+import net.ugurkartal.backend.services.dtos.responses.OrderDetailCreatedResponse;
+import net.ugurkartal.backend.services.dtos.responses.OrderDetailGetAllResponse;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/order-details")
+@RequiredArgsConstructor
+public class OrderDetailController {
+    private final OrderDetailService orderDetailService;
+
+    @GetMapping("/{orderId}")
```

Comment:
for links you don't use camelcase. you can only name it id :)

Reply:
> for links you don't use camelcase. you can only name it id :)

Es handelt sich nicht um einen Link, sondern um einen Variablennamen. orderId steht in geschweiften Klammern :)
Link sieht das aus: /api/order-details/ORD-74748383